bytes/bytes/Device.swift

```diff
@@ -39,104 +39,129 @@ public struct Device {
-            case "iPad8,4":      return "iPad Pro 3rd Gen (11 inch, 1TB, WiFi+Cellular)"
-            case "iPad8,5":      return "iPad Pro 3rd Gen (12.9 inch, WiFi)"
-            case "iPad8,6":      return "iPad Pro 3rd Gen (12.9 inch, 1TB, WiFi)"
-            case "iPad8,7":      return "iPad Pro 3rd Gen (12.9 inch, WiFi+Cellular)"
-            case "iPad8,8":      return "iPad Pro 3rd Gen (12.9 inch, 1TB, WiFi+Cellular)"
+        switch platform {
+        // iPhone
+        case "iPhone1,1":    return "iPhone 1G"
+        case "iPhone1,2":    return "iPhone 3G"
+        case "iPhone2,1":    return "iPhone 3GS"
+        case "iPhone3,1":    return "iPhone 4"
+        case "iPhone3,2":    return "iPhone 4 (Revision A)"
+        case "iPhone3,3":    return "Verizon iPhone 4"
+        case "iPhone4,1":    return "iPhone 4S"
+        case "iPhone5,1":    return "iPhone 5 (GSM)"
+        case "iPhone5,2":    return "iPhone 5 (GSM+CDMA)"
+        case "iPhone5,3":    return "iPhone 5c (GSM)"
+        case "iPhone5,4":    return "iPhone 5c (Global)"
+        case "iPhone6,1":    return "iPhone 5s (GSM)"
+        case "iPhone6,2":    return "iPhone 5s (Global)"
+        case "iPhone7,1":    return "iPhone 6 Plus"
+        case "iPhone7,2":    return "iPhone 6"
+        case "iPhone8,1":    return "iPhone 6s"
+        case "iPhone8,2":    return "iPhone 6s Plus"
+        case "iPhone8,4":    return "iPhone SE"
+        case "iPhone9,1":    return "iPhone 7 (GSM+CDMA)"
+        case "iPhone9,2":    return "iPhone 7 Plus (GSM+CDMA)"
+        case "iPhone9,3":    return "iPhone 7 (Global)"
+        case "iPhone9,4":    return "iPhone 7 Plus (Global)"
+        case "iPhone10,1":    return "iPhone 8 (GSM+CDMA)"
+        case "iPhone10,2":    return "iPhone 8 Plus (GSM+CDMA)"
+        case "iPhone10,3":    return "iPhone X (GSM+CDMA)"
+        case "iPhone10,4":    return "iPhone 8 (Global)"
+        case "iPhone10,5":    return "iPhone 8 Plus (Global)"
+        case "iPhone10,6":    return "iPhone X (Global)"
+        case "iPhone11,2":    return "iPhone XS"
+        case "iPhone11,4":    return "iPhone XS Max (China)"
+        case "iPhone11,6":    return "iPhone XS Max"
+        case "iPhone11,8":    return "iPhone XR"
+        case "iPhone12,1":    return "iPhone 11"
+        case "iPhone12,3":    return "iPhone 11 Pro"
+        case "iPhone12,5":    return "iPhone 11 Pro Max"
             
-            case "i386":         return "Simulator"
-            case "x86_64":       return "Simulator"
-                
-            default: return nil
-            }
+        // iPod
+        case "iPod1,1":      return "1st Gen iPod"
+        case "iPod2,1":      return "2nd Gen iPod"
+        case "iPod3,1":      return "3rd Gen iPod"
+        case "iPod4,1":      return "4th Gen iPod"
+        case "iPod5,1":      return "5th Gen iPod"
+        case "iPod7,1":      return "6th Gen iPod"
+        case "iPod9,1":      return "7th Gen iPod"
```

Comment:
Thanks for your work. I think it would be good to use the same naming conventions for all devices here. So I would suggest append the generation at the end of the name like `"iPad 6th Gen (WiFi)"` in line 127 i.e. `"iPod touch 3rd Gen"`

Reply:
You're welcome! Yes, you are right about that, I changed it.

Reply:
Great, thanks! (Sorry for my late reply)

Reply:
My pleasure!